### DISCUSSION TITLE
New user just some observations

### DISCUSSION THREAD

**Discussion by SinisterSpatula** [DSJAS/DSJAS#11] [General]
Hey there, I'm so impressed by this project, it's amazing and very advanced in the way it works.  Trying to understand it all to get it working for me, and to hopefully be able to design a theme for it.  Just some tips for new users that would have helped me:

If you're going to set up your own Apache/Lamp stack:  the way this project works is a bit cunning - it doesn't just serve up static URL's that point directly at files.  It takes advantage of some re-write tricks so YOU HAVE TO HAVE apache properly configured to do that, and that really confused me at first.  My advice is, take a look at the docker part of the project, the way the dockerfile configures it's apache.  Use that as a guide for how you should configure your own apache if not using docker.  I thought at first that a lot of the pages were not implemented or missing, but once I got my apache properly configured so that the theme can do its URL magic, that got all those missing pages working. :D  Sorry maybe I'm a newb but I didn't grasp that at first, maybe it helps someone =)  If you want to put a note about this in the docs maybe? :D  GREAT WORK ON THIS!  I'm gonna continue learning and understanding it in hopes of adding some more themes/modules.  One that I would like to see is a fake two-factor auth.  I'm guessing I can look at the frustrating log-in to understand how I might be able to do it. :D  

This project as far as fake banks for scambaiting is concerned is by far the most advanced I've seen and I'm just getting started in the hobby.  It's by far the most flexible, customizable, sharable, advanced, unique design I could have ever asked for.  I really hope you continue working on it!!!!!

One thing I'm still not clear on, if I want to add some transactions such as rent payment, utilities, gym membership, groceries etc what's the best way to do that?  It looks like the money has to go from one account to another, but what if I just want it to be a debit?  The money just gets subtracted and a transaction with an amount and a memo shown?   Is there an easy way to do that, or just not implemented yet?  Perhaps there can be "dynamic transactions" that can be populated automatically that use the current date as a basis for it's displayed date, so they always look recent/current, and allow us to supply a list of possible amounts and business types.  EDIT: Oh I see, you do this under force a transaction and there you can select purchase, but there's no easy way to change the date/time of the "purchase" (yet).

**Comment by thediscreetdingus** [comment]
Hi Sinster new user here as well! I've been scambaiting for a while and I've been trying to setup this amazing scam bank for a while but can't seem to get it right. I have my SQL server, Apache server, and everything running on XMAPP. I get to the page where I can either edit as admin or go into the bank and boom bunch of errors. Just not sure if I'm doing something wrong in the install. I wish there was some sort of video documents to help out because I'm a visual learner haha. Anyways I figured I'd reach out to see if you had any interest in helping me out and I don't mind compensating you a bit for your time. Thanks so much in advance. You can contact me on any social media or write a reply here if you have interest. Sorry if this seems abrupt I've literally tried everything and can't get anything to work lol. Thanks again!

**Reply by SinisterSpatula** [reply]
Sure thing id help ya for sure.  Do you have discord?  Add me/message me: SinisterSpatula#1916

**Reply by thediscreetdingus** [reply]
Thank you so much my friend I just added you on Discord. I actually got a bit further but still having trouble loading pages had to edit some file locations as it seems some files were missing from master DL.

**Reply by ejv2** [reply]
Files missing from master is normal, as they are generated by developer scripts. Please follow the instructions in #5 for more information.
